License2Deploy/rolling_deploy.py

```diff
@@ -4,8 +4,8 @@
 import argparse
 from sys import exit, argv
 from time import sleep, time
-from AWSConn import AWSConn
-from set_logging import SetLogging
+from .AWSConn import AWSConn
```

Comment:
Had a problem with running relative imports in python 2.6. This needs to be tested.

Reply:
you are correct, this was an issue initially.
I had to install pkg via pip to resolve vs git clone + execute py directly